template/Makefile.j2

```
@@ -0,0 +1,50 @@
+# =============
+# This file is automatically generated from the templates in stackabletech/operator-templating
+# DON'T MANUALLY EDIT THIS FILE
+# =============
+
+.PHONY: docker chart-lint compile-chart
+
+TAG    := $(shell git rev-parse --short HEAD)
+
+VERSION := $(cargo metadata --format-version 1 | jq '.packages[] | select(.name=="stackable-{[ operator.product_string }]-operator") | .version')
+
+docker:
+	docker build --force-rm -t "docker.stackable.tech/stackable/{[ operator.product_string }]-operator:${TAG}" -t "docker.stackable.tech/stackable/{[ operator.product_string }]-operator:latest" -f docker/Dockerfile .
+	echo "${NEXUS_PASSWORD}" | docker login --username github --password-stdin docker.stackable.tech
+	docker push --all-tags docker.stackable.tech/stackable/{[ operator.product_string }]-operator
+
+## Chart related targets
+compile-chart: version crds config
+
+chart-clean:
+	rm -rf deploy/helm/{[ operator.product_string }]-operator/configs
+	rm -rf deploy/helm/{[ operator.product_string }]-operator/templates/crds.yaml
+
+version:
+	sed -i deploy/helm/{[ operator.product_string }]-operator/Chart.yaml "<version_placeholder>" ${VERSION}
```

```suggestion
        yq --in-place '.version = $ENV.VERSION | .appVersion = $ENV.VERSION' deploy/helm/{[ operator.product_string }]-operator/Chart.yaml
```

The current version fails silently if the command has already been executed.